ui/components/src/ui-components/table/component/Table.tsx

```diff
@@ -1080,6 +1094,33 @@ export class Table extends React.Component<TableProps, TableState> {
       }
 
       className = classnames(className, this.getCellBorderStyle(cellKey));
+      const mergedAdjacentCellsCount = (cellProps.item.mergedCellsCount ?? 1) - 1;
+      let cellWidth = 0;
+
+      for (let i = 0; i <= mergedAdjacentCellsCount; i++) {
+        const col = this.state.columns[index + i];
+        if (this.state.hiddenColumns.indexOf(col.key) === -1) {
+          const gridColumn = gridColumns[index + i - foundHiddenColumns];
+          const gridColumnWidth = gridColumn ? gridColumn.getBoundingClientRect().width : 0;
+          cellWidth += gridColumnWidth;
+        } else {
+          foundHiddenColumns++;
+        }
+
+        if (i > 0)
+          cells[col.key] = (
```

Comment:
this should be wrapped with curly braces

Reply:
Added curly braces